### PR TITLE
167190463 User can highlight an article text and make a comment

### PR DESCRIPTION
**Pull request by cvjude**
#### What does this PR do?
- Should allow a user to highlight an article text and make a comment

#### How can this be manually tested?
- Clone the repo
- Run the command below in your terminal to install all dependencies
```
npm install
``` 
- Run the command below to test
```
npm test
```

#### Any background context you want to provide?
none

#### What are the relevant Pivotal Tracker stories? 
- [#167190463](https://www.pivotaltracker.com/story/show/167190463)

### CODE REVIEW

**Review by topseySuave — APPROVED** [review]

**Review by obayomi96 — APPROVED** [review]
LGTM

**Review by Lundii — APPROVED** [review]
LGTM

File: server/controllers/commentController.js [hunk 1]
```diff
@@ -162,4 +162,42 @@ export default class CommentController {
...
+    if (!highlightedComment) return errorStat(res, 404, 'Post not found');
+    const { highlightUser, articleId, highlightId } = highlightedComment;
+    const newComment = {
+      highlightUser,
+      body: comment,
+      articleId,
+      highlightId,
+      authorId: user.id,
+    };
+    const userComment = await models.Comment.create(newComment);
```
Comment (encodedBicoding) [inline server/controllers/commentController.js:186]: Nice implementation Jude, however, I feel you needed to have added the comment to the post too as it is a child of the post. `await post.addComment(userComment)`. If this is not the case, do you have a relationship where comments can haveMany comments?

Comment (cvjude) [reply server/controllers/commentController.js:186]: The highlight doesn't have a different table. it still uses the comment table.

**Pull request merged**